app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+import requests
+
+
+API_KEY = os.environ.get("API_KEY")
+
+API_URL = f"https://api.weatherapi.com/v1/current.json?key={API_KEY}&q="
+
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print(f"Getting weather for {CITY}")
+    response = requests.get(f"{API_URL}?q={CITY}&key={API_KEY}")
+    if response.status_code == 200:
+        data = response.json()
+        print(f"Weather: {data}")
```

Comment:
You are printing the entire data received from the API. This might not be very useful for the user. It's better to extract the relevant information from the data and present it in a user-friendly way. For example, you could extract the temperature, humidity, wind speed, etc. and print them separately. This is a code quality issue.